src/components/Login.tsx

```diff
@@ -9,49 +9,107 @@ import {
   DialogActions,
   Button,
   TextField,
-  FormControl
-} from "@material-ui/core";
-import ExitToAppIcon from "@material-ui/icons/ExitToApp";
-import { AirDBContext } from "../context/AirDBContext";
+  FormControl,
+  useTheme
+} from '@material-ui/core'
+import { ExitToApp, Edit } from '@material-ui/icons'
+import Question from './take-test/Question'
+import { AirDBContext } from '../context/AirDBContext'
 
 export default React.memo((props: any) => {
-  const { users, updateAirDB } = React.useContext(AirDBContext);
-  const [toggled, setToggled] = React.useState(false);
-  const [username, setUser] = React.useState("");
-  const [password, setPassword] = React.useState("");
+  const { users, updateAirDB } = React.useContext(AirDBContext)
+  const [toggled, setToggled] = React.useState(false)
+  const [username, setUser] = React.useState('')
+  const [password, setPassword] = React.useState('')
+  const [submitted, setSubmitted] = React.useState(false)
+
+  const theme = useTheme()
+
+  const handleSubmitionClick = (newUsername: string, newPassword: string) => {
+    setSubmitted(true)
+    // TODO: more stuff
+  }
 
   const handleClose = () => {
-    setToggled(false);
-  };
+    setToggled(false)
+  }
 
   const handleLogin = () => {
     const isMatching =
       users.filter(
         user =>
           user.fields.password === password && user.fields.username === username
-      ).length > 0;
-    const userId = users.filter(user => user.fields.username === username)[0]
-      .id;
+      ).length > 0
+    const userId = users.filter(user => user.fields.username === username)[0].id
 
     if (isMatching) {
-      setToggled(false);
-      props.setRedirect("/");
-      window.localStorage.setItem("username", username);
-      updateAirDB("Testy - Users", userId, {
-        active: "true"
-      });
+      setToggled(false)
+      props.setRedirect('/')
+      window.localStorage.setItem('username', username)
+      updateAirDB('Testy - Users', userId, {
+        active: 'true'
+      })
     }
-  };
+  }
 
   return users.length > 0 ? (
     <>
       <br />
       <Paper>
-        <Typography variant="h4">Home</Typography>
-        I'd put a lot of text here...{" "}
-        <IconButton onClick={() => setToggled(true)} color="primary">
-          <ExitToAppIcon />
-        </IconButton>
+        <Typography variant="h3">Home</Typography>
+        <div
+          style={{
+            padding: '1em',
+            border: '2px solid',
```

Comment:
You can combine your `border color` here by doing `border: 2px solid theme.palette.primary.main`